ClosedXML_Tests/Excel/CalcEngine/MathTrigTests.cs

```diff
@@ -955,19 +947,18 @@ public void Floor(double input, double significance, double expectedResult)
             Assert.AreEqual(expectedResult, actual, tolerance);
         }
 
-        [TestCase(6.7, -1)]
-        public void Floor_ThrowsNumberExceptionOnInvalidInput(double input, double significance)
-        {
-            Assert.Throws<NumberException>(() => XLWorkbook.EvaluateExpr($"FLOOR({input.ToInvariantString()}, {significance.ToInvariantString()})"));
-        }
-
         [TestCase(6.7, 0)]
         [TestCase(-6.7, 0)]
         public void Floor_ThrowsDivisionByZeroOnZeroSignificance(double input, double significance)
         {
             Assert.Throws<DivisionByZeroException>(() => XLWorkbook.EvaluateExpr($"FLOOR({input.ToInvariantString()}, {significance.ToInvariantString()})"));
         }
 
+        [TestCase(6.7, -1)]
+        public void Floor_ThrowsNumberExceptionOnInvalidInput(double input, double significance)
+        {
+            Assert.Throws<NumberException>(() => XLWorkbook.EvaluateExpr($"FLOOR({input.ToInvariantString()}, {significance.ToInvariantString()})"));
+        }
```

Comment:
Here and after `Atan2_ReturnsMinus3QuartersOfPiWhenFirstSmaller0AndSecondItsNegative` the empty lines between methods are missing.